config/config/src/test/java/io/helidon/config/KeyTokenResolvingTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2021 Oracle and/or its affiliates.
+ * Copyright (c) 2017, 2022 Oracle and/or its affiliates.
```

Comment:
Only copyright updated on this file

Reply:
Sorry for the force-push, had to do the squash